x-pack/plugins/apm/public/components/app/ServiceMap/Popover/ServiceStatsList.tsx

```diff
@@ -50,15 +52,17 @@ export function ServiceStatsList({
     },
     {
       title: i18n.translate(
-        'xpack.apm.serviceMap.avgReqPerMinutePopoverMetric',
+        'xpack.apm.serviceMap.avgTransactionRatePopoverMetric',
         {
-          defaultMessage: 'Req. per minute (avg.)',
+          defaultMessage:
+            'Trans. per {unit, select, minute {m} second {s}} (avg.)',
```

Comment:
This makes it say "Trans. per s (avg.)" would be better as:

```suggestion
            'Trans. per {unit, select, minute {minute} second {second}} (avg.)',
```

Reply:
or maybe even just "Trans. rate" since the whole thing is "Trans. per s (avg.) | 0.07 tps"